Resources/Prototypes/XenoArch/Effects/normal_effects.yml

```diff
@@ -418,6 +418,23 @@
   components:
   - type: IgniteArtifact
 
+- type: artifactEffect
+  id: EffectGravityWell
+  targetDepth: 3
+  effectHint: artifact-effect-hint-environment
+  components:
+  - type: GravityWellArtifact
```

Comment:
just add the gravity well component here instead of adding it inside of the system.

Reply:
Done.